### PR TITLE
Update dataset API usage in distill lstm

### PR DESCRIPTION
**Pull request by LiuChiachi**
### PR types

Others
### PR changes

Models
### Describe

dataset重新设计，模型需要更新，目前均已自测通过

### CODE REVIEW

**Comment by ZeyuChen** [comment]
@LiuChiachi ChnSentiCrop dataset upgrade commit has already merged.

**Comment by LiuChiachi** [comment]
> @LiuChiachi ChnSentiCrop dataset upgrade commit has already merged.

File: examples/model_compression/distill_lstm/utils.py [hunk 1]
```diff
@@ -98,70 +101,27 @@ def convert_example(example,
...
         # Get the label
-        label = example[-1]
-        example = example[:-1]
-        # Create label maps if classification task
-        if label_list:
-            label_map = {}
-            for (i, l) in enumerate(label_list):
-                label_map[l] = i
-            label = label_map[label]
+        label = example['labels'] if 'labels' in example else example['label']
```
Comment (LiuChiachi) [inline examples/model_compression/distill_lstm/utils.py:109]: 求问下， @ZeyuChen @smallv0221 ，glue 系列dataset的key是`'sentence'`和`labels'`, ChnSentiCorp的key是`'text'`和`'label'`，需要统一下吗。如果不需要的话，我这里的写法好吗

File: examples/model_compression/distill_lstm/data.py [hunk 2]
```diff
@@ -98,31 +92,43 @@ def _data_augmentation(data, whole_word_mask=whole_word_mask):
...
                 if new_text not in used_texts:
-                    new_data.append([words, data[1]])
-                    used_texts.add(new_text)
-    train_dataset.data.data = new_data
+                    new_data.append({
+                        "sentence": words,
+                        "labels": data['labels']
+                    })
+            used_texts.add(new_text)
+    train_dataset.new_data = new_data
```
Comment (ZeyuChen) [inline examples/model_compression/distill_lstm/data.py]: @smallv0221 Review the preprocessing pipeline here. Is it possible not to expose `new_data` variable in this example? @LiuChiachi

File: examples/model_compression/distill_lstm/data.py [hunk 3]
```diff
@@ -249,7 +268,7 @@ def create_distill_loader(task_name,
         vocab = tokenizer
         pad_val = tokenizer.pad_token_id
 
-    if task_name == 'senta':
+    if task_name == 'chnsenticorp':
         train_ds = apply_data_augmentation_for_cn(
```
Comment (ZeyuChen) [inline examples/model_compression/distill_lstm/data.py]: Is it possible to use the `dataset.map()` to do augmentation? @smallv0221 Discuss it with @LiuChiachi

**Review by ZeyuChen — CHANGES_REQUESTED** [review]
Need further review since it expose the `new_data` variable, it's very confusing for new user.

File: examples/model_compression/distill_lstm/data.py (hunk 3 above)
Comment (smallv0221) [reply examples/model_compression/distill_lstm/data.py]: yes

Comment (LiuChiachi) [reply examples/model_compression/distill_lstm/data.py]: Yes. Done. Thanks for your suggestions

File: examples/model_compression/distill_lstm/README.md [hunk 4]
```diff
@@ -163,18 +168,19 @@ CUDA_VISIBLE_DEVICES=0 python bert_distill.py \
...
+    --embedding_name w2v.google_news.target.word-word.dim300.en \
 
 ```
 
 各参数的具体说明请参阅 `args.py` ，注意在训练不同任务时，需要调整对应的超参数。
 
 
 ## 蒸馏实验结果
-本蒸馏实验基于GLUE的SST-2、QQP、中文情感分类ChnSentiCorp数据集。实验效果均使用每个数据集的验证集（dev）进行评价，评价指标是准确率（acc），其中QQP中包含f1值。利用基于BERT的教师模型去蒸馏基于Bi-LSTM的学生模型，对比Bi-LSTM小模型单独训练，在SST-2、QQP、senta(中文情感分类)任务上分别有3.3%、1.9%、1.4%的提升。
+本蒸馏实验基于GLUE的SST-2、QQP、中文情感分类ChnSentiCorp数据集。实验效果均使用每个数据集的验证集（dev）进行评价，评价指标是准确率（acc），其中QQP中包含f1值。利用基于BERT的教师模型去蒸馏基于Bi-LSTM的学生模型，对比Bi-LSTM小模型单独训练，在SST-2、QQP、chnsenticorp(中文情感分类)任务上分别有3.3%、1.9%、1.4%的提升。
```
Comment (ZeyuChen) [inline examples/model_compression/distill_lstm/README.md]: chnsenticorp -> ChnSentiCorp

Comment (ZeyuChen) [reply examples/model_compression/distill_lstm/README.md]: 数据集官方名是这个

Comment (ZeyuChen) [reply examples/model_compression/distill_lstm/README.md]: 就改README文档就好，其他地方不用改

**Review by ZeyuChen — APPROVED** [review]
LGTM @smallv0221 You do the final review then merge code.

File: examples/model_compression/distill_lstm/README.md (hunk 4 above)
Comment (LiuChiachi) [reply examples/model_compression/distill_lstm/README.md]: Done. Thanks

**Review by smallv0221 — APPROVED** [review]
LGTM
